src/read/stream.rs

```diff
@@ -27,13 +26,16 @@ impl<R: Read> ZipStreamReader<R> {
         let central_header_start = 0;
 
         // Parse central header
-        let signature = self.0.read_u32_le()?;
-        if signature != spec::CENTRAL_DIRECTORY_HEADER_SIGNATURE {
-            Ok(None)
-        } else {
-            central_header_to_zip_file_inner(&mut self.0, archive_offset, central_header_start)
-                .map(ZipStreamFileMetadata)
-                .map(Some)
+        match ZipEntryBlock::parse(&mut self.0) {
+            Err(_) => Ok(None),
```

Comment:
Can we just pass along the error in this case, simplifying the return type to `ZipResult<ZipStreamFileMetadata>`?

Reply:
Sure!! I will fix the tests to do that.

Reply:
Done! No tests changed!